source/style-guide/style/code-examples.txt

```diff
@@ -14,6 +14,23 @@ Code Examples
 Use the following guidelines when creating blocks of code as input
 or output examples:
 
+-  Code examples should be formatted and checked to ensure they are idiomatic
+   using software tooling. Integrations with most IDEs and Editors are available.
```

Comment:
Nitpick: `editors` should be a normal noun here since nothing is being specifically named.